api/nvidia/v1/clusterpolicy_types.go

```diff
@@ -1872,6 +1931,15 @@ func (p *DevicePluginSpec) IsEnabled() bool {
 	return *p.Enabled
 }
 
+// IsEnabled returns true if IMEX DRA Driver is enabled through gpu-operator
+func (d *IMEXDRADriverSpec) IsEnabled() bool {
+	if d.Enabled == nil {
+		// default is true if not specified by user
+		return true
```

Comment:
You've made it enabled by default. Does it mean that IMEX is common/expected to be widely used?